### PR TITLE
Add a disclaimer about author's mindset on top of README.

### PR DESCRIPTION
**Pull request by Thesourtimes**
Congratulations, your effort have even more ruined lives of millions and will likely be mentioned in Degree-level history books of future if our grand children won't be slaved by a Dengist/Duginist tyranny, I don't even want to dig into what a disgrace you are in FOSS community and an example of why you shouldn't be doing such stupid things in public for pragma.

Even if you've deleted that code, your mindset is most likely still intact and users of this module has the right of knowing It and be advised to look at alternatives.

Sincerely,
Kuna

### CODE REVIEW

**Review by kalix123 — APPROVED** [review]

**Review by Soltra45 — APPROVED** [review]

**Comment by kalix123** [comment]
Hey Brandon, You should absolutely merge this. Maybe change up the language a bit. But you could probably face legal repercussions if you don't warn people about what they are downloading.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+⚠️ DISCLAIMER
+===
+
+DO NOT use or update this module if you or your production servers etc. are located in Belarus. Author of this package forgot the diffrence between activism and vandalism and put a malicious code to wipe out data and put "❤️" on where they were. You are warned about this sick mindseet. Use on your own risk.
```
Comment (oscar-legorretam) [inline README.md:4]:
```suggestion
DO NOT use or update this module if you or your production servers etc. are located in a country actively involved in the invasion of Ukraine. Author of this package opposes said invasion and put a malicious code to wipe out data and put "❤️" on where they were. You are warned about this feature. Use on your own risk.
```

**Comment by oscar-legorretam** [comment]
Added a suggested new legend that could make for a good warning.

**Comment by thepwrtank18** [comment]
image
image

**Review by Turtgun — APPROVED** [review]

**Review by artur6768 — APPROVED** [review]

**Review by SoulRepo — APPROVED** [review]

**Review by Zxilly — APPROVED** [review]
Take responsibility

**Comment by thepwrtank18** [comment]
merging is still blocked

**Comment by ThamognyaKodi** [comment]
This should be added.

**Review by thepwrtank18 — APPROVED** [review]

**Review by swampservers — APPROVED** [review]

**Review by jonathan-rosa — APPROVED** [review]

**Review by s0menickname — APPROVED** [review]

**Review by Nauxscript — APPROVED** [review]

**Review by ThamognyaKodi — APPROVED** [review]

**Review by vt-idiot — APPROVED** [review]

**Review by BrandonHuu — APPROVED** [review]

**Review by Coolsonickirby — APPROVED** [review]

**Review by DillonPotts — APPROVED** [review]

**Review by jerbear2008 — APPROVED** [review]

**Comment by zlw9991** [comment]
we need to @ Dengists/Duginists to weigh in on the situation

**Review by initpwn — APPROVED** [review]

**Review by initpwn — APPROVED** [review]

**Review by initpwn — APPROVED** [review]
LGTM

**Review by jukrb0x — APPROVED** [review]
TLDR version:

eat my shit

**Review by zlw9991 — APPROVED** [review]

**Review by dave0null — APPROVED** [review]
